src/promptflow/promptflow/_utils/logger_utils.py

```diff
@@ -267,11 +267,15 @@ def update_log_path(log_path: str, input_logger: logging.Logger = None):
     if input_logger:
         logger_list.append(input_logger)
     for logger_ in logger_list:
-        for wrapper in logger_.handlers:
-            if isinstance(wrapper, FileHandlerConcurrentWrapper):
-                handler: FileHandler = wrapper.handler
-                if handler:
-                    wrapper.handler = type(handler)(log_path, handler._formatter)
+        update_single_log_path(log_path, logger_)
+
+
+def update_single_log_path(log_path: str, logger_: logging.Logger):
```

Comment:
It may not be a private method. Currently, all logs printed by different loggers (logger, flow_logger, bulk_logger...) will be in one file. But for batch run, if the user wants to pay attention to the execution progress, he only needs to view the logs of bulk logger. Therefore, we add this new function to provide the ability to modify the path of a logger individually, so we can put the logs printed by the bulk logger into another file.

Related issue: https://github.com/microsoft/promptflow/issues/519
You can consider whether to implement this feature or not.

Reply:
We can do this thing when updating docstring in the next PR.